backend/tracim_backend/command/cleanup.py

```diff
@@ -280,8 +280,8 @@ def _delete_user_database_info(
             print('all user "{}" revisions deleted'.format(user.user_id))
 
         if should_anonymize.need_anonymization and not force_delete_all_associated_data:
-            cleanup_lib.delete_user_associated_data(user)
             cleanup_lib.safe_delete(user.config)
+            cleanup_lib.delete_user_associated_data(user)
```

Comment:
Can you explain just briefly why previous code cause dry run to fail ?

Reply:
dry-run does an `expunge()` when an object is "updated", so the user object is not part of any session after the `delete_user_associated_data(user)` call.

Reply:
In this case should we add a simple test for dry-run that just verify the command doesn't failed ?